chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -359,12 +359,19 @@ def find_node_boot_mac_addresses(node, admin_data_net)
           else
             raise "Unsupported version of Windows Server / Hyper-V Server"
           end
+          if os =~ /^hyperv/
```

Comment:
Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.